bin/sdk/notifications.js

```diff
@@ -0,0 +1,38 @@
+/** @format */
+/**
+ * External dependencies
+ */
+const HtmlWebpackPlugin = require( 'html-webpack-plugin' );
+const path = require( 'path' );
+const spawnSync = require( 'child_process' ).spawnSync;
+
+exports.config = ( { argv: { outputDir }, getBaseConfig, __rootDir } ) => {
+	const baseConfig = getBaseConfig();
+
+	return {
+		...baseConfig,
+		...{
```

Comment:
nit: `...{` can be omitted, specify the extra properties directly.

Reply:
removed in 381a0563d6